source/docs/installation.md

```diff
@@ -0,0 +1,79 @@
+title: Installation
+---
+
+## Prerequisites
+
+In order to make Embark work in your computer, we need to have some tools installed first. Make sure you have the following ready and in the correct version:
+
+- [Node](#Node)
+- [IPFS](#IPFS-Optional)
+- [Ethereum Client](#Ethereum-Client-Optional)
+
+Once done, go ahead and [install Embark](#Installing-Embark).
+
+### Node
+
+Please install [Node.js](http://nodejs.org/) in version 8.11.3 LTS or higher.
+
+{% note info Quick tip: %}
+**We recommend installing Node using the [Node Version Manager](https://github.com/creationix/nvm/blob/master/README.md).**  This is because it makes it very easy to install different versions of Node in isolated environments that don't require users to [change their permissions](https://docs.npmjs.com/getting-started/fixing-npm-permissions) when installing packages. Find instructions on how to install NVM [here](https://github.com/creationix/nvm/blob/master/README.md#install-script).
+
+Once that is done, you can install and select a specific Node version using the `nvm` command like this:
+
+<pre><code class="shell">$ nvm install 8.11
```

Comment:
maybe we should swap `8.11` for `--lts` — valid for `nvm install` and `nvm use` commands.

Reply:
I had no idea this option existed. Thanks, that's much better indeed :)